wercker.yml

```diff
@@ -213,17 +213,15 @@ build:
           echo "build is skipped ..."
         fi
 
-  # disabled until https://github.com/checkstyle/checkstyle/issues/7534
-  #
-  # - script:
-  #     name: NoExceptiontest - Checkstyle ,sevntu-checkstyle javadoc
-  #     code: |
-  #       if [[ $RUN_JOB == 1 ]]; then
-  #         echo "Command: ./.ci/wercker.sh no-exception-checkstyle-sevntu-javadoc"
-  #         ./.ci/wercker.sh no-exception-checkstyle-sevntu-javadoc
-  #       else
-  #         echo "build is skipped ..."
-  #       fi
+   - script:
+       name: NoExceptiontest - Checkstyle ,sevntu-checkstyle javadoc
```

Comment:
please move this update to some other PR or to separate commit.

Reply:
This is commit, https://github.com/checkstyle/checkstyle/pull/7816/commits/efd44b6a926c648295154bcab7875b50fc2420f3, is separate from https://github.com/checkstyle/checkstyle/pull/7816/commits/0133923af75ab4edbd49e08f6bf106c7066d8993, but would you like me to switch it to another PR?

Reply:
This commit should not reference pmd issue 7810. Please use prefix " Issue 7534: ....", And keep it this PR